File: exo2_3.py
import os
import sys
nb_proc = 5
for i in range(nb_proc):
    pid_f = os.fork()
    if pid_f != -1:
        if pid_f == 0:
            print("Fils")
            break
        else:
            print("Père")
    else:
        print("Erreur(création du fils)")

File: exo2_3.c
#include <stdio.h>
#include <unistd.h>
#include <sys/wait.h>

int main () {
	int i;
	for (i = 0; i < 5; i++)
	{
		if (fork() == 0) {
			printf ("Fils %d créé\n", i + 1);
			return 0;
		}
	}
	for (i = 0; i < 5; i++)
	{
		for (i = 0; i < 5; i++)
			wait(NULL);
	}
	return 0;
}
